e2e/wait-for-frontend.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+import fetch from 'node-fetch';
+import delay from 'delay';
+
+export default async function() {
```

Comment:
What does this file do? I don't think I understand it :(

Reply:
right now, it has an error I just realized, due to my forgetting to import `config`.

What it does is try to load the frontend website and retry one second later if it fails. It returns immediately on success. It's a way to make sure the e2e tests are never launched before frontend is running.